examples/nextjs/next-env.d.ts

```diff
@@ -2,4 +2,4 @@
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
```

Comment:
deliberate?

Reply:
happened automatically

Reply:
seems incorrect though, the new one redirects to the existing one

Reply:
i guese next was one of the packages updated and that comes with this change for some reason, i'm just gonna leave it